girder/models/setting.py

```diff
@@ -279,6 +279,12 @@ def validateCoreEmailFromAddress(doc):
             raise ValidationException('Email from address must not be blank.', 'value')
 
     @staticmethod
+    @setting_utilities.validator(SettingKey.CONTACT_EMAIL_ADDRESS)
+    def validateCoreContactEmailAddress(doc):
+        if not doc['value']:
+            raise ValidationException('Email from address must not be blank.', 'value')
```

Comment:
Per [the discussion here](https://stackoverflow.com/questions/8147805/the-mailto-setting-a-proper-name-on-the-recipient) we probably want to support more than a simple email address regex (especially since the default `EMAIL_FROM_ADDRESS` default already uses a proper name component).

The space and URL encoding described by the RFC also should be done, but probably not in the validator (as that will make the value hard to edit). The best place is probably in the Pug template.